documentation/how-to-guides/deploy_magma_access_gateway.md

```diff
@@ -0,0 +1,59 @@
+# How-to: Deploy Magma Access Gateway using magma-access-gateway snap
+
+The goal of this document is to guide the Operator through the process of deploying 
+Magma Access Gateway to a [compliant](#System requirements) piece of hardware using 
+[magma-access-gateway](https://snapcraft.io/magma-access-gateway) snap.<br>
+Unless specified otherwise, all actions in this guide are executed using `root` account.
+
+### System requirements
+
+- 64bit-X86 machine
+- Baremetal strongly recommended
+- Two ethernet interfaces (SGi and S1)
+- Internet connectivity from SGi interface
+- Ubuntu 20.04 LTS installed
+  ([Ubuntu installation guide](https://help.ubuntu.com/lts/installation-guide/amd64/index.html))
+
+## 1. Install magma-access-gateway snap
+
+From your Magma AGW host machine execute:
+
+```bash
+snap install magma-access-gateway --classic
```

Comment:
Should we add `--edge` to the install command?

Reply:
So I didn't put it deliberately - if you don't put it, command will fail and you'll be instructed to use `--edge` and in short time we should have package published to other channels, so `--edge` will not be necessary. So essentially it's all about not having to update the docs after we promote snap.